### PR TITLE
[NSO-1808] Updates to playbook to reflect live code

### PR DESCRIPTION
**Pull request by aheadley**

### CODE REVIEW

File: tasks/ssl.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+
+- name: Create SSL key file
+  copy:
+    dest: /root/mellon-ssl.key
```
Comment (clwells) [inline tasks/ssl.yml]: quote

File: tasks/ssl.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    content: "{{ vault_mellon_ssl_key }}"
+    owner: root
```
Comment (clwells) [inline tasks/ssl.yml]: quote

File: tasks/ssl.yml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    group: root
```
Comment (clwells) [inline tasks/ssl.yml]: quote

File: tasks/ssl.yml [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    mode: '0640'
```
Comment (clwells) [inline tasks/ssl.yml]: remove quotes

File: tasks/master.yml [hunk 5]
```diff
@@ -1,15 +1,13 @@
 ---
+
 - name: Install the master server
-  shell: ipa-server-install -r {{ ansible_domain | upper }} -n {{ ansible_domain }} -p {{ directory_admin_password }}  -U -a {{ kerberos_admin_password }} --ip-address {{ ansible_default_ipv4.address }}
+  shell: >
+    ipa-server-install -U \
```
Comment (clwells) [inline tasks/master.yml]: '\' not needed.  Do 1 arg per line.

File: tasks/master.yml [hunk 6]
```diff
@@ -1,15 +1,13 @@
...
+      -r {{ freeipa_domain | upper }} -n {{ freeipa_domain }} \
+      -p {{ directory_admin_password | quote }} -a {{ kerberos_admin_password | quote }} \
+      --hostname {{ inventory_hostname }} --ip-address {{ frontnet_addr }} \
+      -N --setup-dns --auto-forwarders --forward-policy only
   args:
     creates: /etc/ipa/default.conf
```
Comment (clwells) [inline tasks/master.yml]: quote

File: tasks/master.yml [hunk 7]
```diff
@@ -1,15 +1,13 @@
...
-
-# TODO requires EL 7.2
-- name: Open firewall for replication
-  firewalld:
-    service: freeipa-replication
-    permanent: true
-    state: enabled
-    immediate: yes
-  when: "(ansible_distribution == 'Fedora' or ansible_distribution_major_version == '7') and {{ freeipa_replicas |length }} > 0"
+  become: yes
```
Comment (clwells) [inline tasks/master.yml]: true instead of yes

File: tasks/replica.yml [hunk 8]
```diff
@@ -1,40 +1,14 @@
...
-
-- name: Decode the base64 file
-  shell: base64 -d {{ replica_file }}.base64 > {{ replica_file }}
-  args:
-    creates: "{{ replica_file }}"
 
 - name: Install the replica
-  shell: ipa-replica-install -U -p {{ directory_admin_password }}  -w {{ kerberos_admin_password }} {{ replica_file }}
+  shell: >
+    ipa-replica-install -U \
```
Comment (clwells) [inline tasks/replica.yml]: no '\', 1 arg per line. quote {{ }}

File: tasks/master.yml (hunk 5 above)
Comment (clwells) [reply tasks/master.yml]: quote {{ }}

File: tasks/server.yml [hunk 9]
```diff
@@ -1,23 +1,20 @@
...
-  when: ansible_fqdn == freeipa_master
+  when: inventory_hostname == freeipa_master
 
 - include: replica.yml
-  when: ansible_fqdn in freeipa_replicas
+  when: inventory_hostname in freeipa_replicas
 
-# TODO make backups
+- include: ssl.yml
+  become: yes
```
Comment (clwells) [inline tasks/server.yml]: true

File: tasks/server.yml [hunk 10]
```diff
@@ -1,23 +1,20 @@
...
-  firewalld:
-    service: freeipa-ldap
-    permanent: true
-    state: enabled
-    immediate: yes
-  when: ansible_distribution == 'Fedora' or ansible_distribution_major_version == '7'
+  with_items:
+    - ipa-server
+    - ipa-server-dns
+  become: yes
```
Comment (clwells) [inline tasks/server.yml]: true

File: tasks/ssl.yml [hunk 11]
```diff
@@ -0,0 +1,22 @@
...
+  register: ssl_key_file
+
+- name: Create SSL cert file
```
Comment (clwells) [inline tasks/ssl.yml]: same as above re: quoting/no-quotes

**Comment by aheadley** [comment]
@clwells Updated and ready for re-review

**Pull request merged**